mmv1/api/type.rb

```diff
@@ -266,6 +266,14 @@ def lineage
       "#{__parent.lineage}.#{name&.underscore}"
     end
 
+    # Prints the access path of the field in the configration eg: metadata.0.labels
+    # The only intended purpose is to get the value of the labes field by calling d.Get().
+    def access_path
```

Comment:
nit: Maybe `terraform_lineage` / `lineage_terraform` given these do largely the same thing?

Reply:
Will change it to `terraform_lineage`